CONTRIBUTING.md

```diff
@@ -212,6 +213,11 @@ fuzzed. Fuzzers are build when `cmake` is passed the `-DBUILD_FUZZER=ON`
 flag, they use [libFuzzer](http://llvm.org/docs/LibFuzzer.html). Anything
 that parses input from the network is a good candidate for fuzzing.
 
+# Mruby modules
+
+It is possible to add additional mruby modules.
+All modules that match `Dir.glob("../mruby-*/mrbgem.rake")` get linked. Copy (or git-submodule) the library under `deps/` and rerun make.
```

Comment:
Could we add a link to https://github.com/h2o/h2o/wiki/Importing-third-party-code as the recommended method of adding this to `deps`?

At least for the things that gets merged to master, we would like to have the code dumped in a traceable way.